apps/mobile/src/components/Markdown.tsx

```diff
@@ -46,6 +49,8 @@ type GalleryMarkdownProps = PropsWithChildren<{
   style?: StyleProp<unknown>;
 }>;
 
+const KNOWN_NON_DEEPLINK_ROUTES = ['community', '~', 'home', 'token'];
```

Comment:
imo this kind of deeplink-related logic should live in `DeepLinkRegistrar.tsx`, and `Markdown.tsx` should remain as "dumb" as possible